tfprotov6/provider.go

```diff
@@ -147,6 +170,26 @@ type GetProviderSchemaResponse struct {
 	Diagnostics []*Diagnostic
 }
 
+// GetResourceIdentitySchemasRequest represents a Terraform RPC request for the
+// provider's resource identity schemas.
+type GetResourceIdentitySchemasRequest struct{}
+
+// GetResourceIdentitySchemasResponse represents a Terraform RPC response containing
+// the provider's resource identity schemas.
+type GetResourceIdentitySchemasResponse struct {
+	// ResourceSchemas is a map of resource names to the schema for the
```

Comment:
I missed this in the other PR 😄 , I believe v5 has the same mistype
```suggestion
	// IdentitySchemas is a map of resource names to the schema for the
```

Reply:
Good catch! Yeah, v5 had the same one – I just added an commit to this PR to fix that while we're at it